constantine/math/elliptic/ec_twistededwards_batch_ops.nim

```diff
@@ -115,9 +115,12 @@ func batchInvert*[F](
     dst[i] *= accumulator
     accumulator *= elements[i]
 
-func batchInvert*[F](dst: openArray[F], source: openArray[F]): bool {.inline.} =
+func batchInvert*[F](dst: var openArray[F], source: openArray[F]): bool {.inline.} =
   if dst.len != source.len:
     return false
   let N = dst.len
   batchInvert(dst.asUnchecked(), source.asUnchecked(), N)
   return true
+
+func batchInvert*[N: static int, F](dst: array[N, F], src: array[N, F]) =
+  batchInvert(dst.asUnchecked(), src.asUnchecked(), N)
```

Comment:
this will not compile at the moment because the inner `batchInvert` currently returns a bool

Reply:
line 126 `batchInvert` takes in unchecked array as input, i.e this calls the function at line 90